Lib/sqlite3/dbapi2.py

```diff
@@ -55,16 +55,24 @@ def TimestampFromTicks(ticks):
 collections.abc.Sequence.register(Row)
 
 def register_adapters_and_converters():
+    from warnings import warn
```

Comment:
Would it also be possible/desirable to display a warning when a SQLite connection is initialized with the `detect_types` parameter, which (if I recall correctly) activates the default converters?

Reply:
Yeah, that is possible. OTOH, the user may already have deregistered the default converters, so such a warning may feel spammy in such situations. We could detect that, though. Maybe it is worth the added complexity.

Reply:
I think it's fine to just warn here.

Reply:
I do too. See my latest post on the issue. Thanks for bringing it up though.